lib/liquid/standardfilters.rb

```diff
@@ -421,6 +421,7 @@ def at_most(input, n)
 
     def default(input, default_value = ''.freeze)
       if !input || input.respond_to?(:empty?) && input.empty?
+        Usage.increment("liquid.default_filter_received_false_value") if input == false # See https://github.com/Shopify/liquid/issues/1127
```

Comment:
Wouldn't we end up with a different metric name for each type of usage?  That means that we would need to add each new metric to whatever dashboard graph that is tracking these.  Instead, why don't we pass `"default_filter_received_false_value"` as the name to `Usage.increment` then we can use this as a tag instead of as the metric name in our implementation of `Usage.increment`.  Having a static metric name (e.g. `liquid.usage`) with a tag for the usage type would avoid the need to update the dashboard to add a new type of usage.

Reply:
Yeah makes sense. I'll tweak this a little bit and may as well drop the tag support, if needed in the future It can be added then